docs/conceptual/core/consensus.md

```diff
@@ -128,12 +128,12 @@ finality. This article provides a brief overview of the offerings included with
 
 ### Aura
 
-[Aura](https://crates.parity.io/substrate_consensus_aura/index.html) provides a slot-based block
+[Aura](https://substrate.dev/rustdocs/master/pallet_aura/index.html) provides a slot-based block
```

Comment:
This link should go to the crate that the outer node uses to actually run consensus, rather than the pallet.
```suggestion
[Aura](https://substrate.dev/rustdocs/master/sc_consensus_aura/index.html) provides a slot-based block
```

Reply:
Thank you!